RecoMET/METFilters/python/ecalEEBadCalibSep2017ListFilter_cfi.py

```diff
@@ -0,0 +1,28 @@
+import FWCore.ParameterSet.Config as cms
+ 
+ecalBadCalibSep2017ListFilter = cms.EDFilter(
+  "EcalBadCalibFilter",
+ 
+  # use this if using AOD:
+  # the Ecal rechit collection found in AOD
+  EcalRecHitSource = cms.InputTag('reducedEcalRecHitsEE'),
+ 
+ 
+  # use this if using MINIAOD:
+  # the Ecal rechit collection found in MINIAOD
+  # EERecHitSource = cms.InputTag('reducedEgamma','reducedEERecHits'),
+ 
+ 
+ 
+  # minimum rechit et to flag as bad: 
+  ecalMinEt        = cms.double(50.),
+  # DetId of bad channel:
+  baddetEcal        = cms.vuint32(872439604,872422825,872420274,872423218,
+                                  872423215,872416066,872435036,872439336,
+                                  872420273,872436907,872420147,872439731,
+                                  872436657,872420397,872439732,872439339,
+                                  872439603),                                  
+  taggingMode = cms.bool(False),
```

Comment:
hi @dvannerom @slava77 - should this not be "True"?

Reply:
Dear David, I believe the tagging modes of the MET filters are set to false by default, i.e. the analyzers using them in MINIAOD usually just want those events out of their data. When using those filters for efficiency studies, as we do in the MET group, the taggingMode is set to True in the global configuration file and overwrites this command.

Reply:
the METFilters machinery implies that the filter actually selects or fails events on a corresponding filter path. So, it has to be in the filtering (not in tagging) mode.

Reply:
ok- thanks the the explanation. I see how this works now.